### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
### Building core functionality: a calculator

**In this PR: 👇**

- Delete all the boilerplate from CRA (text, images, styles);
- Create a new Calculator.js file;
- Create a React component that matches the given design;
- Use class-based components;

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "extends": ["airbnb", "plugin:react/recommended"],
     "plugins": ["react"],
     "rules": {
+      "linebreak-style": 0,
```
Comment (Dipeshtwis) [inline .eslintrc.json]: - [ ] Please don't edit the `eslintrc` file. These are the standard which needs to be maintained between all the `Micronauts` and make your code better.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @elyor-doniyorov, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Removed all the boilerplate that got created by the CRA.
:+1: Used class-based component. Well Done.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json (hunk 1 above)
Comment (elyor-doniyorov) [reply .eslintrc.json]: Hello Dipesh,

Thank you for your review. But I am a bit confused. I reached out to you on slack. Please check it

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @elyor-doniyorov 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer. 
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**